lib/utils/config.ts

```diff
@@ -60,7 +60,7 @@ export function generateBaseConfig(
 		deviceType?: string;
 		os?: {
 			sshKeys?: string[];
-		}
```

Comment:
These cosmetic amendments to `config.ts` were done by `npm run prettify`, which is automatically called by `npm test`.